packages/webapp/src/components/Finances/Transaction/ExpandedContent/styles.module.scss

```diff
@@ -43,10 +43,9 @@ button.toDetail {
   flex-flow: column nowrap;
   justify-content: center;
   align-items: center;
-  background: #f6fbfa;
 
   .tabs {
-    margin: 24px 0 24px 0;
+    margin: 4px 0px 24px 0px;
```

Comment:
It looks as though I'm breaking the rule of 8 but it appears there was some margin/padding supplied by (I think) the table itself... pretty sure (at least by visual check) this ends up being 8px overall.